gems/quilt_rails/README.md

```diff
@@ -829,10 +834,28 @@ The `Performance::Reportable` mixin uses [https://github.com/Shopify/statsd-inst
 
 ### Generators
 
-#### `quilt:install`
+#### `quilt_rails:install`
+
+Run both generator below.
+
+##### `quilt_rails:sewing_kit:install`
+
+Adds a basic `sewing-kit.config.ts` file and basic linting, format configuration.
+This generator assume the use of [`sewing-kit`](https://github.com/Shopify/sewing-kit)
+
+##### `quilt_rails:quilt:install`
+
+Run all the generator below.
+
+###### `quilt_rails:quilt:rails_setup`
+
+Mounts the Quilt engine in `config/routes.rb` and provide a Procfile to kick start node in production container.
```

Comment:
The doc should use `node` or `Node`.  I prefer `Node`, but I don't mind `node` so long as we're consistent.